atomic_reactor/buildimage.py

```diff
@@ -112,6 +113,7 @@ def get_reactor_tarball_path(self, tmpdir):
         os.chdir(local_reactor_git_path)
         try:
             logger.debug("executing sdist command in directory '%s'", os.getcwd())
+            sys.stderr.flush()
```

Comment:
My initial thought was "only flush when we have to", rather than always flushing, thinking there could be a performance impact, but that does seem a lot more fragile with how many seems like they might be necessary, so centralizing it is probably the way to go. I'll adjust accordingly.